aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -301,6 +301,23 @@ struct TORCH_CUDA_CPP_API CTCLossDescriptor
 #endif
 };
 
+struct TORCH_CUDA_CPP_API ActivationDescriptor
```

Comment:
Cool!! We should be able to use this in the future to support other activations as well https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnActivationMode_t.